{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2022-29820",
      "ASSIGNER": "security@jetbrains.com"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-668"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://www.jetbrains.com/privacy-security/issues-fixed/",
          "name": "https://www.jetbrains.com/privacy-security/issues-fixed/",
          "refsource": "MISC",
          "tags": [
            "Vendor Advisory"
          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In JetBrains PyCharm before 2022.1 exposure of the debugger port to the internal network was possible"
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:jetbrains:pycharm:*:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:A/AC:L/Au:N/C:P/I:N/A:N",
        "accessVector": "ADJACENT_NETWORK",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "NONE",
        "availabilityImpact": "NONE",
        "baseScore": 3.3
      },
      "severity": "LOW",
      "exploitabilityScore": 6.5,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2022-04-28T10:15Z",
  "lastModifiedDate": "2022-05-05T14:33Z"
}